pennylane_lightning/src/Gates.cpp

```diff
@@ -556,6 +556,52 @@ void Pennylane::CSWAPGate::applyKernel(const StateVector& state, const std::vect
 
 // -------------------------------------------------------------------------------------------------------------
 
+const string Pennylane::Unitary::label = "QubitUnitary";
+
+Pennylane::Unitary Pennylane::Unitary::create(const vector<double>& parameters) {
+    auto size = parameters.size();
```

Comment:
Probably not use case for now since we are binding PennyLane, though might be worth verifying the size of `parameters`.